README.md

```diff
@@ -10,9 +10,8 @@
 
 # csaf_distribution
 
-An implementation of a [CSAF](https://csaf.io/)
-[2.0 Spec](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html)
-([Errata](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html))
+Implements a [CSAF](https://csaf.io/)
+([Specification v2.0](https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html))
```

Comment:
Hi @fvsamson, nice to read from you! :)

The problem here is that https://docs.oasis-open.org/csaf/csaf/v2.0/csaf-v2.0.html only links to the errata and there is no easily found link to the current main content of the 2.0 speficiation. It is there, but under appendix A of the errata. Also the new link to the main specification https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html, which you have deleted in this change, does not link to the errata. So your change makes it much harder to access both. 

Any idea to resolve this for good? What about making a new section that explains the standard and the link both with more explanation?

Reply:
Fixed IMO.  Please re-review.